package.json

```diff
@@ -15,6 +15,8 @@
     "snyk": "^1.290.1"
   },
   "dependencies": {
+    "cypress": "^6",
+    "cypress-file-upload": "^5",
```

Comment:
These should be dev dependency (or we can rely on FF, it dynamically installs Cypress)